src/components/reusable/nav/mod.rs

```diff
@@ -52,6 +53,28 @@ pub fn Nav(cx: Scope<Props>) -> Element {
             loop {
                 let list = multipass.list_incoming_request().unwrap_or_default();
                 if list.len() != *reqCount.get() {
+                    // If list is updated, we update the request count
+                    if list.len() > *reqCount.get() {
+                        // We display a notification if the list length is increased (incoming request appended).
+                        // We do not display a notification if the list length is decreased (incoming request is rejected).
+
+                        // TODO: Display sender name of incoming friend request
+                        let most_recent_friend_request = list.last().unwrap().from(); // This returns a FriendRequest struct
```

Comment:
Wouldnt advice unwrapping because if it panics for whatever reason (which it would if the list didnt contain anything), it would kill the task. What i would advice doing is using a `if let` or `match` condition or maybe using `list.is_empty()` to see if its not empty and proceed from there

Reply:
Removed the notification description thing for now, thank you for this advice tho!